pkg/utils/BUILD

```diff
@@ -0,0 +1,8 @@
+package(default_visibility = ["//pkg:__subpackages__"])
```

Comment:
We try to discourage "utils" packages if we can (as per normal Go conventions). Can this functionality all be put into a different package with a more descriptive name, or spread out into different existing packages?